client/checkout/blocks/payment-method-label.js

```diff
@@ -18,23 +18,29 @@ export default ( {
 	const bnplMethods = [ 'affirm', 'afterpay_clearpay', 'klarna' ];
 
 	// Stripe expects the amount to be sent as the minor unit of 2 digits.
-	const amount = normalizeCurrencyToMinorUnit(
-		cartData.totals.total_price,
-		cartData.totals.currency_minor_unit
+	const amount = parseInt(
+		normalizeCurrencyToMinorUnit(
+			cartData.totals.total_price,
+			cartData.totals.currency_minor_unit
+		),
+		10
 	);
 
 	// Customer's country or base country of the store.
 	const currentCountry =
 		cartData.billingAddress.country ||
-		window.wcBlocksCheckoutData.storeCountry;
+		window?.wcBlocksCheckoutData?.storeCountry ||
```

Comment:
```suggestion
		window.wcBlocksCheckoutData?.storeCountry ||
```

Could there be an instance where `window` is not defined?